app/views/govuk_component/tasklist.raw.html.erb

```diff
@@ -0,0 +1,84 @@
+<%
+  steps ||= false
+  small ||= false
+  heading_level ||= 2
+  open_section ||= false
+  remember_last_section ||= false
+
+  section_count = 0
+%>
+<% if steps %>
+  <div
+    data-module="tasklist"
+    class="pub-c-tasklist js-hidden <% if not small %>pub-c-tasklist--large<% end %>"
+    <% if remember_last_section %>data-remember<% end %>
+    >
+    <ol>
+      <% steps.each_with_index do |step, index| %>
+        <%
+          step_active = false
+          step.each do |this_step|
+            if this_step[:panel_links].to_a.any? { |s| s[:active] == true }
+              step_active = true
+            end
+          end
+        %>
+        <li class="pub-c-tasklist__step <% if step_active %>pub-c-tasklist__step--active<% end %>" <% if step_active %>aria-current="step"<% end %>>
```

Comment:
(I personally prefer removing the need for `end` like so: `class="<%= 'pub-c-tasklist__step--active' if step_active' %>"`)

Reply:
Again, my Ruby knowledge. That's helpful, thanks. I'm definitely learning..